pkgs/development/tools/tabnine/default.nix

```
@@ -24,6 +24,7 @@ in stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
+    chmod +x $src
     ln -s $src $out/bin/TabNine
```

```suggestion
    install -Dm755 $src $out/bin/TabNine
```
This should be easier. Please test.

---

it works, thanks